src/index.ts

```diff
@@ -1,4 +1,26 @@
-//Uncomment to see the choose pokemon page
-//const newTemplate = document.querySelector('#choose-page-template')!;
-//const app = document.querySelector('#app')!;
-//app.innerHTML = newTemplate.innerHTML
\ No newline at end of file
+import { showAPopUpScreen } from './showAPopUpScreen';
+
+const app = document.querySelector('#app') as HTMLDivElement
```

Comment:
it would be better to place it inside of a function, to make this `scoped`. We don't want to have this constant to be available everywhere, just while rendering (in renderApp function).
Aaand then there would be need to pass it to functions